client/src/featureform/register.py

```diff
@@ -332,6 +332,21 @@ def __init__(self, registrar, provider):
     def name(self) -> str:
         return self.__provider.name
 
+class StoreProvider:
```

Comment:
Do we want this as StoreProvider? Maybe FilestoreProvider instead for clarity

Reply:
good idea. I was thinking about the right term. It's something that offers offline and online storage. Perhaps hybrid store?